code/modules/mining/lavaland/weather.dm

```diff
@@ -0,0 +1,110 @@
+var/global/datum/weather/activeWeather
```

Comment:
It was gonna be bad/laggy with turfs but since it's area based we can probably have tons of weather